packages/@uppy/utils/src/getDroppedFiles/utils/webkitGetAsEntryApi/index.js

```diff
@@ -25,17 +25,17 @@ function getAsFileSystemHandleFromEntry (entry, logDropError) {
   }
 }
 
-async function* createPromiseToAddFileOrParseDirectory (entry) {
+async function* createPromiseToAddFileOrParseDirectory (entry, relativePath) {
   // For each dropped item, - make sure it's a file/directory, and start deepening in!
   if (entry.kind === 'file') {
     const file = await entry.getFile()
     if (file !== null) {
-      file.relativePath = getRelativePath(entry)
+      file.relativePath = relativePath ? `${relativePath}/${entry.name}` : null
```

Comment:
Can we use `??` here instead?

Reply:
You only can if we would take `relativePath` as is, since we don't, it has to be a ternary.